### ISSUE TITLE
Review Language: Hebrew (Python)

### ISSUE THREAD

**Issue by madiedgar** [legesher/legesher-translations#171]
## Review Hebrew Translations for Python

In order to review this language to the project, complete the steps detailed in the [translation guide](https://docs.legesher.io/legesher-translations/translation-guide) (from Step #6 onward). We are primarily working developing Legesher for the Python programming language. In the future we hope to add more programming languages in the future.

**Reviewing** a translation is extremely important as we want to best understand the culture and context behind the words we select as keywords for the programming languages. The more perspectives the better! 👀Please take your time in being thorough with these translations. Think back when you were learning how to program and how context of words help you bridge together meaning and functionality.

> ⚠️ Note: this issue is to review the Python keywords and builtin functions only. Any additional translation reviews are encouraged, but not necessary.

**To complete this issue**:

- [x] Update any content in this issue from `LANGUAGE` to the name of the language. (Romanian, Arabic, Welsh etc.)
- [x] Title this issue "Review Language: Hebrew (Python)"
- [ ] Update the Python section of the Hebrew's `locale/__.yml` file (files are named using the language abbreviation) by adding comments as followed.

  - **Agreeable:** Leave a `# @username ✅_with any additional commentary afterwards_` after a translation if you agree that the right word to give context to the keyword is the one initially translated.
  - **Optimistic:** Leave a `# @username ❓_with any additional commentary afterwards_` after a translation if you think there may be a better translation / form of the translation
  - **Unsure:** Leave a `# @username 🤔_with any additional commentary afterwards_` after a word that you're not sure if there is a translation

  This issue will be complete when all the words for a programming language looks something like this:

  ```yml
  int: "initiatorsTranslation" # ✅ @firstreviewer, ❓@secondreviewer what about this otherTranslation, ❓@thirdreviewer I agree with otherTranslation, ✅ @fourthreviewer, ✅ @fifthreviewer I also think otherTranslation would be a good option
  ```

- [ ] Merge changes into branch titled `hebrew-translation` on this repo, **NOT** the `master` branch. Please reference this issue in the pull request.

### Reviewers Needed

In order to add the translations for official use across Legesher, the translations will need to be approved and accepted by **5 additional native / fluent speakers**. If you would like to be one of the reviewers for this language, please mention in this issue! :smile: Also feel free to loop in a friend that might be able to help with the translations! Please refer to the [translation guide](https://docs.legesher.io/legesher-translations/translation-guide) for the next step in the review process.

**Review Status**:

- [x] `Add New Dialect` Issue (#144) 
- [x] Initiator: @Adarwall (#149) 
- [ ] Reviewer 1: 
- [ ] Reviewer 2:
- [ ] Reviewer 3:
- [ ] Reviewer 4:
- [ ] Reviewer 5:

**Comment by Gal-Gilor** [comment]
Hi, I'm a native speaker, and I wish to contribute to this project. 
* _Full disclosure: I'm a beginner when it comes to contributing to open-source projects_

**Comment by madiedgar** [comment]
That's wonderful @Gal-Gilor! I've assigned you to this issue. The [translation guide](https://docs.legesher.io/legesher-translations/translation-guide) should be helpful in understanding the step-by-step process and a great place to start. Please reach out if you need help along the way. 👍🏾

**Comment by Gal-Gilor** [comment]
@madiedgar,

I hope the pull request I made is correct.
I added my reviewer comments to the Python section. 

I accidentally also added the reviews to the ATOM section.
For now, I left it as is. I assumed someone would need to review that in the future anyway :)

Please let me know if everything is okay.

**Comment by michalirak** [comment]
@madiedgar
Hi, also a native speaker and would like to contribute

**Comment by madiedgar** [comment]
That would be awesome @michalirak ! I'll assign you to this issue, and you can get started. Let me know if you have any questions. 👍🏾

**Comment by TomerPacific** [comment]
Hey @madiedgar, looking to help. Is this still relevant?

**Comment by madiedgar** [comment]
Yes! Would love your help @TomerPacific. Let me know if you have any questions!

**Comment by TomerPacific** [comment]
@madiedgar - this issue states to update only the Python section. Would you like me to just go over what was reviewed already and fill in what is missing? Or should I add translations to other languages as well?

**Comment by TomerPacific** [comment]
@madiedgar - I have opened a PR. I checked out from the hebrew-translation branch. It seems like it isn't updated with the master branch. Seeing as there are conflicts, would you like me to do this merge?

**Comment by madiedgar** [comment]
Merged your PR @TomerPacific 👍 Once the Python keywords are officially reviewed for the translation we'll merge it and sync it with master. Your PR was great!

**Comment by TomerPacific** [comment]
@madiedgar - glad to be of service.  Would you like me to open an issue for another language? What would be your preferred coding language?

**Comment by madiedgar** [comment]
That would be awesome @TomerPacific 👍  If you'd like to keep reviewing translations, I'd go with Javascript and Atom's editor keywords. After Legesher Python is ready for use, we'll begin working on extending those capabilities to Javascript within Atom's text editor. Feel free to open new issues for both Javascript and Atom. Thank you!

**Comment by madiedgar** [comment]
Happy October @TomerPacific, @Gal-Gilor, @michalirak, and @Adarwall , thank you for your early support on this project! I'm excited about adding the Hebrew translation to Legesher, but I could use your assistance in gathering enough reviewers. Would you consider reaching out to your friends or peers who share an interest and invite them to join us? 😄 Thank you!

**Comment by TomerPacific** [comment]
@madiedgar  - How many more reviews are you looking for?

**Comment by madiedgar** [comment]
@TomerPacific We only need two more reviewers for the Python keywords and builtin functions, so we're very close!
